user/build-stages.md

```diff
@@ -97,6 +97,25 @@ jobs:
 ```
 {: data-file=".travis.yml"}
 
+### Naming your Jobs within Build Stages
+
+You can also name specific jobs within build stages. We recommend unique job names, but 
+do not enforce it (though this may change in the future). Job names are defined like other 
+job configuration elements within a build stage, but the value of the `name` must be in quotes.
+Here's an example of a build using build stages and job names:
+
+```yaml
```

Comment:
this should probably include the top-level key `jobs` (or `matrix`) shouldn't it?

Reply:
+1

Reply:
whoops! yes - thanks